rasa/cli/test.py

```diff
@@ -267,7 +266,7 @@ def run_core_test(args: argparse.Namespace) -> None:
     Args:
         args: the parsed CLI arguments for 'rasa test core'.
     """
-    asyncio.run(run_core_test_async(args))
+    asyncio_run_workaround(run_core_test_async(args))
```

Comment:
@ka-bu Why do we need this?

Reply:
we needed it because we had trouble with the version of uvloop then

Reply:
okay, so we can revert that? sweet